SECURITY.md

```diff
@@ -1,55 +1,52 @@
-# A note on the security of the included implementations and schemes
+# PQClean Security Notice
 
-This project contains (reference) implementations of cryptographic libraries.
-We do not make any security claims about the code included in PQClean.
-In the current state, we distribute reference implementations with minor modifications.
-We did not perform any extensive security analyses.
-This code is suitable for experimental or scientific use.
-We recommend careful expert code review before using any of the included implementations in production environments.
+## Project Scope
 
-See also the [NIST PQC Forum][forum] for discussion about the cryptographic schemes included in PQClean.
+PQClean offers reference implementations of cryptographic algorithms, primarily for academic and experimental purposes. 
+It is important to understand that these implementations have not been subjected to rigorous security audits or formal security validations. 
 
-## Current and past security issues
+## Security Considerations
 
-We will attempt to document security vulnerabilities made known to us on a best-effort basis.
-If an issue is marked with a date, the issue has been resolved since any commits made after that date.
+Users should be aware that while PQClean aims to provide accurate and secure implementations, the project does not make explicit security claims. 
+Currently, the code is best suited for research purposes and users should assess its suitability for their specific needs, especially in 
+security-sensitive environments. Any use in a production environment should be preceded by a detailed expert security review. 
 
-Again, we emphasise that the code in this repository has not seen any formal analysis or audit!
-Use at your own risk.
+## Vulnerability Reporting
 
-### Historic issues
-* LEDAcryptKEM `leaktime` implementations are known to not be constant-time and expected to have timing side channel vulnerabilities.
-  * LEDA support has since been dropped from PQClean as it got eliminated from the NIST PQC standardization project in round 3.
-* Rainbow level I has been shown to be insecure by Ward Beullens.
-  * Rainbow was eliminated from the NIST PQC standardization project and removed from PQClean in round 4.
+We endeavor to document known security vulnerabilities as they are brought to our attention. Resolved issues will be clearly marked with the date of resolution. 
```

Comment:
You're dropping some important information here, as we don't have release dates people should be paying attention to commits.

Reply:
Actually, when a security vulnerability is resolved, it is marked into the SECURITY.md file with the date of its resolution. This is evident from the original SECURITY.md file contents, where each fix is clearly marked with a date of resolution. The wording here does not refer to release dates, but to the dates in the SECURITY.md file. If you want to change the logic by which security fixes are tagged, I am all for it, just that we don't miscommunicate about the contents of the SECURITY.md file.